src/core/Song.cpp

```diff
@@ -608,6 +608,9 @@ void Song::playPattern( const Pattern* patternToPlay, bool loop )
 
 void Song::updateLength()
 {
+	if(m_loadingProject) {
```

Comment:
```cpp
if(m_loadingProject)
{
```

Reply:
I think either of them is fine, but notice the space between `if` and `(`.
```suggestion
	if (m_loadingProject) {
```
```suggestion
	if (m_loadingProject)
	{
```

Reply:
> I think either of them is fine, but notice the space between `if` and `(`.

It's actually in the [coding convention](https://github.com/LMMS/lmms/wiki/Coding-Conventions#braces) but I'm really happy with either one. You're second suggestion would be the correct one. The rest of the file adheres to the brackets on separate lines. I guess this will be run over by a code formatting script some day anyway.

Reply:
The coding convention allows it, so I compacted the if clause into one line.